hi-ml-cpath/testhisto/testhisto/utils/test_viz_utils.py

```diff
@@ -167,15 +166,15 @@ def test_plot_attention_tiles_below_min_rows(slide_node: SlideNode, caplog: LogC
     slide_node.bottom_tiles = []
     with caplog.at_level(logging.WARNING):
         bottom_tiles_fig = plot_attention_tiles(
-            case="TP", slide_node=slide_node, top=False, num_columns=4, figsize=(10, 10)
+            case="FN", slide_node=slide_node, top=False, num_columns=4, figsize=(10, 10)
         )
```

Comment:
if I understand well lots of these tests have been designed to check the plots are created without issues, rather than checking the content of the plot itself. 

My suggestion would be to add a few more cases only for ` test_plot_slide` below where we check the binary files actually match - that means also the text in them is what we expect

Reply:
Done! I combined the cases in the same test cause plots were taking ~1.5s on my machine